src/components/views/settings/Notifications.tsx

```diff
@@ -417,8 +418,10 @@ export default class Notifications extends React.PureComponent<IProps, IState> {
                 }
             }
             const kind = PushRuleKind.ContentSpecific;
+            const DOT = "DOT";
```

Comment:
would using this string replacement suffice?
Or should I put a validation on fullstops?

Reply:
the variable would need to be lowercase named (and I don't think it even needs to be a variable?), but we should also use characters that are unlikely to be used by the user's existing rules.